app/controllers/spree/admin/errors_controller.rb

```diff
@@ -1,11 +1,17 @@
 module Spree
   module Admin
     class ErrorsController < BaseController
+
       skip_before_action :authorize_admin
 
       def forbidden
+        authorize! :read, ::Spree::AdminPanel
```

Comment:
I don't think we should define `Spree::AdminPanel` in spree_core, and then use it only in spree_backend. Maybe a cleaner version would be checking whether there's a `:read` permission on `Spree::Admin` (which is already a module defined in spree_backend)?
https://github.com/CanCanCommunity/cancancan/blob/develop/docs/define_check_abilities.md#can-subjects

Reply:
Also, maybe this authorize could be moved to the `Admin::BaseController`, and we could handle the redirection to 404 page manually there?